tests/Search.js

```diff
@@ -11,18 +11,20 @@ describe("Search", () => {
   it("Search with valid word", async () => {
     await (await Menu.searchButton).waitForDisplayed();
     await (await Menu.searchButton).click();
-    const text = "Finish The Most Iconic Book Titles In History";
-    await (await Menu.searchButton).setValue(text);
+    const validTextToSearch = "Finish The Most Iconic Book Titles In History";
+    await (await Menu.searchButton).setValue(validTextToSearch);
     await (await Menu.searchIcon).waitForDisplayed();
     await (await Menu.searchIcon).click();
-    await expect(await Menu.searchResultBoxMessage).toHaveTextContaining(text);
+    await expect(await Menu.searchResultBoxMessage).toHaveTextContaining(
+      validTextToSearch
+    );
   });
 
   it("Search with invalid word", async () => {
     await (await Menu.searchButton).waitForDisplayed();
     await (await Menu.searchButton).click();
-    const text = "qwq";
-    await (await Menu.searchButton).setValue(text);
+    const validTextToSearch = "qwq";
```

Comment:
On the description of the step, you specify that it is tested the invalid word, but the name of the variable suggests something else.